controllers/grid/navigationMenus/form/NavigationMenuForm.inc.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/**
+ * @file controllers/grid/navigationMenus/form/NavigationMenuForm.inc.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class NavigationMenuForm
+ * @ingroup controllers_grid_navigationMenus_form
+ * @see NavigationMenu
+ *
+ * @brief Form for manager to create/edit NavigationMenus.
+ */
+
+
+import('lib.pkp.classes.form.Form');
+
+class NavigationMenuForm extends Form {
+	/** @var int Context ID */
+	var $_contextId;
+
+	/** @var $navigationMenu NavigationMenu The menu being edited */
+	var $_navigationMenuId;
+
+	/**
+	 * Constructor
+	 * @param $contextId int Context ID
+	 * @param $navigationMenuId int NavigationMenu Id
+	 */
+	function __construct($contextId, $navigationMenuId = null) {
+		$this->_navigationMenuId = !empty($navigationMenuId) ? (int) $navigationMenuId : null;
```

Comment:
The `(int)` cast should be happening in the DAO, so IMO there's no point cluttering up assignments in form classes with additional casts/checks.